CHANGES.md

```diff
@@ -1,5 +1,17 @@
 # Changelog
 
+## [0.2.1] - 2020-12-07
+### Fixed
+- Upgrade to `colombe` version `0.4.0`, this fixes vanishing full stop issue,
```

Comment:
Shouldn't this be `0.5.0`?

Reply:
No, this is old release commit we merged back to master. Next release should probably be `0.3.0` or `0.2.2`.

Reply:
Ok, we should release those soon, so there is no desync between the versions in the changelog and the versions we use.